docs/semgrep-ci/running-semgrep-ci-without-semgrep-app.md

```diff
@@ -55,11 +55,7 @@ Running Semgrep in CI without Semgrep App bears specific limitations compared to
 
 * Stand-alone CI jobs cannot send [PR or MR comments](/semgrep-app/notifications/#enabling-github-pull-request-comments). These comments describe the finding and help developers resolve vulnerabilities and other code issues.
 * Stand-alone CI jobs cannot block a pull or merge request based on a finding. There is no distinction or tiering between rules. Alternatively, Semgrep App's [Rule Board](/semgrep-app/rule-board) can tier rules and undertake actions, such as blocking PRs or MRs on a finding generated by a reliable rule.
-* Findings are dumped to a log, so there is no record of a finding's **triage state**, such as opened, closed, or ignored.
-
-:::note
-`semgrep ci` prevents the duplication of findings by scanning only the changes in files in pull or merge requests. This is different from Semgrep App's capability to track a finding's **triage state**, as each finding has a record.
```

Comment:
It is true, I just thought it was confusing since I don't really see diff scanning as a way to prevent duplicate findings, so much as suppress irrelevant ones. But feel free to add back!